src/components/Upload.vue

```diff
@@ -1,66 +1,63 @@
       v-btn(v-if="closeable", icon, flat, @click="$emit('close')")
         v-icon close
-  .px-0.py-2
-    slot(name="header")
-      .title.px-3
-        v-avatar.mr-2(color="blue-grey lighten-4")
-          v-icon {{ ResourceIcons[dest._modelType.toUpperCase()] }}
-        span {{ dest.name }}
 
   slot(name="dropzone")
     .dropzone-wrapper(
         v-if="!files.length", :class="dropzoneClass", @dragenter="dropzoneClass = 'animate'",
         @dragleave="dropzoneClass = null", @drop="dropzoneClass = null")
       .dropzone-message
         v-icon(size="50px") attach_file
-        .title.mt-3 {{ dropzoneMessage }}
+        .title {{ dropzoneMessage }}
       input.file-input(type="file", :multiple="multiple", @change="filesChanged")
 
-  .pb-2.px-3(v-show="files.length && !errorMessage")
-    div(v-show="!uploading")
-      v-subheader {{ statusMessage }}
-      v-btn(color="warning", @click="files = []")
-        v-icon.mr-1 close
-        | Clear all
-      v-btn(color="success", @click="start")
-        v-icon.mr-1 play_arrow
-        | Start upload
-
-  div(v-if="errorMessage")
-    v-alert(:value="true", type="error")
-      span.body-2.mr-2 {{ errorMessage }}
-      v-btn(v-if="!uploading", color="purple darken-1", dark, @click="start")
-        v-icon.mr-1 replay
-        | Resume upload
-
-  slot(name="progress")
-    div(v-if="uploading")
-      .subheading.px-3.
-        {{ formatSize(totalProgress) }} / {{ formatSize(totalSize) }} ({{ totalProgressPercent }}%)
-      v-progress-linear(:value="totalProgressPercent", height="20")
-
-  slot(name="files")
-    v-list.file-list.pb-4(v-show="files.length", dense)
-      v-list-tile.file-tile(v-for="(file, i) in files", :key="file.file.name", avatar,
-          :class="`status-${file.status}`")
-        v-list-tile-avatar
-          v-btn.mx-0(v-if="file.status === 'pending'", icon, @click="files.splice(i, 1)")
-            v-icon close
-          v-progress-circular(v-if="file.status === 'uploading'", color="primary", :rotate="-90",
-              :value="progressPercent(file.progress)", :indeterminate="file.progress.indeterminate")
-          v-icon(v-if="file.status === 'done'", color="success", large) check
-          v-icon(v-if="file.status === 'error'", color="error", large) warning
-        v-list-tile-content
-          v-list-tile-title {{ file.file.name }}
-          v-list-tile-sub-title
-            span(v-if="file.progress.current") {{ formatSize(file.progress.current ) }} /&nbsp;
-            span {{ formatSize(file.file.size) }}
+  v-card
+    div(v-show="files.length && !errorMessage")
+      div(v-show="!uploading")
+        v-subheader {{ statusMessage }}
+        v-btn(@click="files = []") Clear all
+        v-btn(color="primary", @click="start") Start upload
+
+    div(v-if="errorMessage")
+      v-alert(:value="true", type="error") {{ errorMessage }}
+        v-btn(v-if="!uploading", color="purple darken-1", dark, @click="start")
+          v-icon replay
```

Comment:
Right, will do.

Reply:
It appears that the alert content doesn't look vertically aligned properly (more space at the top than bottom. Vuetify's alert component usually has a darker 'border' element at the top. Is that missing from here (it could be why it doesn't look middle aligned to me)?

<img width="1133" alt="screen shot 2018-09-19 at 1 41 05 pm" src="https://user-images.githubusercontent.com/1560779/45770921-b9a71180-bc11-11e8-8994-b57c8a6e9c0d.png">